File: Algorithms/fizzBuzz.js
function fizzbuzz(num){
  if(num <= 0){
    console.log("Paramater must be a positive number");
  }
  else {
    for(var i = 1; i <= num; i++){
    if(i%3==0 && i%5==0){
      console.log("FizzBuzz");
    } else if(i%3==0 && i%5!=0){
      console.log("Fizz");
    } else if(i%5==0 && i%3!=0){
      console.log("Buzz");
    } else{
      console.log(i);
    }
    }
  }
}

fizzbuzz(0);

File: java/javaFun/FizzBuzz/FizzBuzz.java
public class FizzBuzz {
	public static String fizzBuzz(int number) {
		String result = "None";
		if(number == 3) {
		    result = "Fizz";
		} else if(number == 5) {
		    result = "Buzz";
		} else if (number == 15){
		    result = "FizzBuzz";
		} else if (number == 2){
			result = "" + 2;
		}
		return result;
	}
}
